### PR TITLE
Add feature expense design

### PR DESCRIPTION
**Pull request by mayankvishwakarma012**
**Linked Work Items**
-https://internship-splitwise.atlassian.net/jira/software/projects/IS/issues/IS-69

**Checklist**
- [x] Code adheres to **[Git and Pull Request Guidelines]**
- [x] Code adheres to **[Angular Guidelines]**
- [x] Code adheres to **[Front-end Checklist]**

### CODE REVIEW

File: src/app/expense/add-edit-expence/add-edit-expence.component.html [hunk 1]
```diff
@@ -1 +1,48 @@
...
+
+        <div class="form-group mt-2">
+            <label for="participants">Participants:</label>
+            <input type="text" class="form-control" id="participants" name="participants" required>
+        </div>
+
+        <button type="submit" class="btn btn-primary">Add Expense</button>
+        <button type="button" class="btn btn-secondary">Cancel</button>
+    </form>
+</main>
```
Comment (ravipromact) [inline src/app/expense/add-edit-expence/add-edit-expence.component.html:65]: Add comments, I should not remind this everytime

File: src/app/expense/view-expense-list/view-expense-list.component.html [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+    </li>
+</ul>
+
+<div class="modal fade" id="addExpenseForm" tabindex="-1">
+    <div class="modal-dialog ">
+        <div class="modal-content ">
+            <app-add-edit-expence></app-add-edit-expence>
+        </div>
+    </div>
+</div>
```
Comment (ravipromact) [inline src/app/expense/view-expense-list/view-expense-list.component.html:27]: Add comments

File: src/app/group/add-edit-group/add-edit-group.component.html [hunk 3]
```diff
@@ -5,9 +5,9 @@
     <div class="modal-header">
 
         <!-- if clicked on add/ edit  -->
-        <h1 class="modal-title fs-5" id="exampleModalLabel">{{formHeading}}</h1>
+        <h1 class="modal-title fs-5" id="Form-heading">{{formHeading}}</h1>
```
Comment (ravipromact) [inline src/app/group/add-edit-group/add-edit-group.component.html]: Don't use capitalize word in #id selector Form-heading

File: src/app/group/view-group-list/view-group-list.component.html [hunk 4]
```diff
@@ -33,16 +33,29 @@ <h3>Members:</h3>
...
 
         <!-- expenses of this roup will be displayed here -->
-        <div class="card card-body">
-            <h3>Expenses:</h3>
+        <div class="card card-body" data-bs-toggle="collapse" data-bs-target="#collapseExpense" routerLink="/dashboard/group/expense">
+            <div class="row">
+                <div class="col">
+                    <h3 routerLink="/dashboard/group/expense" data-bs-toggle="collapse" data-bs-target="#collapseExpense">Expenses:</h3>
+                </div>
+                <div class="col text-end">
```
Comment (ravipromact) [inline src/app/group/view-group-list/view-group-list.component.html:43]: Add comments

File: src/app/group/view-group-list/view-group-list.component.ts [hunk 5]
```diff
@@ -1,12 +1,48 @@
...
+        }
+      });
+
+
+
+  }
+
+
+
+  }
```
Comment (ravipromact) [inline src/app/group/view-group-list/view-group-list.component.ts:41]: Add comments

File: src/app/group/view-group-list/view-group-list.component.ts [hunk 6]
```diff
@@ -1,12 +1,48 @@
...
+
+          for (let group of this.groups ){
+            if(this.currentUrl == '/dashboard/group/view/'+group){
+            this.isGroupActive = true;
+            break;
+          }
+        }
+      });
+
+
```
Comment (ravipromact) [inline src/app/group/view-group-list/view-group-list.component.ts]: remove unnecessary empty space

File: src/app/expense/add-edit-expence/add-edit-expence.component.html (hunk 1 above)
Comment (mayankvishwakarma012) [reply src/app/expense/add-edit-expence/add-edit-expence.component.html:65]: Resolved.

File: src/app/expense/view-expense-list/view-expense-list.component.html (hunk 2 above)
Comment (mayankvishwakarma012) [reply src/app/expense/view-expense-list/view-expense-list.component.html:27]: Resolved.

File: src/app/group/add-edit-group/add-edit-group.component.html (hunk 3 above)
Comment (mayankvishwakarma012) [reply src/app/group/add-edit-group/add-edit-group.component.html]: Resolved.

File: src/app/group/view-group-list/view-group-list.component.html (hunk 4 above)
Comment (mayankvishwakarma012) [reply src/app/group/view-group-list/view-group-list.component.html:43]: Resolved.

File: src/app/group/view-group-list/view-group-list.component.ts (hunk 5 above)
Comment (mayankvishwakarma012) [reply src/app/group/view-group-list/view-group-list.component.ts:41]: Resolved.

File: src/app/group/view-group-list/view-group-list.component.ts (hunk 6 above)
Comment (mayankvishwakarma012) [reply src/app/group/view-group-list/view-group-list.component.ts]: Resolved.

**Pull request merged**